test/README.md

```diff
@@ -124,12 +124,6 @@ and somewhat self-explanatory.
     stack test --test-arguments="--pattern foo042 --accept"
     ```
 8. Check the content of `expected`. Maybe the test didn't do what you thought it would. Fix and go back to 7 until it's ok.
-9. Add under `Extra-source-files` in `idris.cabal` the patterns that match the folder's content. If you forget this, your test will fail in Travis CI. With the previous example, it should be at least:
```

Comment:
`Extra-source-files` appears to be empty right now. It has a comment that suggests that it's auto-generated:
```
-- extra-source-files is generated by Setup.hs using `git --ls-files`.
Extra-source-files:
```
Moreover, the last update to `test/README.md` also seemed to remove mention of `Extra-source-files`: https://github.com/idris-lang/Idris-dev/commit/3285d369fda50cb94e1a894e642649ba441a6615#diff-c30ff793110b0960f76bb3854af6f5a8

Am I missing something?